### DISCUSSION TITLE
Need help debugging ' STATUS_FAILED' on add_playlist_items()

### DISCUSSION THREAD

**Discussion by TychoZaal** [sigma67/ytmusicapi#427] [Q&A]
Hi, when running my self-made Python script to convert Spotify playlists to YTM nothing happens. Upon debugging I found out that I get **STATUS_FAILED** as a response. The exact same program used to work up until a few weeks ago. The playlistId seems good, and the list of videoIds is also filled with valid Ids. Is there any way to figure out why this process has failed, as I would love to resolve this! 

I provide the method with a **playlistId** (retrieved from user library), a list of strings of song titles-> **songs**, and the **playlistName**

_def update_youtube_music_playlist(playlistId, songs, playlistName):_

    ytmusicTracks = []
    progress = 0
    songAmount = len(songs)

    for song in songs:
        videoId = ytmusic.search(query=song, filter="songs", limit=1)[0]["videoId"]
        ytmusicTracks.append(videoId)
        progress = progress + 1
        print("Updating", playlist, ":", progress, "/", songAmount, "\r", end="")

    response = ytmusic.add_playlist_items(playlistId=playlistId, videoIds=ytmusicTracks)
    print("Updated the " + playlist + " playlist")`

The _response_ Dictionary contains a status variable with **STATUS_FAILED** as a value

**Comment by sigma67** [comment]
First off, you can probably just use this: https://github.com/sigma67/spotify_to_ytmusic

If the script used to work you should probably set up your credentials from scratch. Maybe you signed out in the browser you created them from. I recommend using oauth, those credentials are not as prone to expiry.
